### PR TITLE
[@types/dragula] Implemented slideFactorX and slideFactorY parameters for enhanced dragging control

### PR DESCRIPTION
**Pull request by ValerioEmanuele**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://www.npmjs.com/package/dragula>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "definitely-typed",
-    "version": "0.0.3",
+    "version": "0.0.4",
```
Comment (andrewbranch) [inline package.json:4]: Please revert this change. It’s not related to the @types packages inside the repo.

```suggestion
    "version": "0.0.3",
```

**Review by andrewbranch — CHANGES_REQUESTED** [review]
Looks good except for the package.json change

**Review by andrewbranch — APPROVED** [review]

File: types/dragula/v2/index.d.ts [hunk 2]
```diff
@@ -17,6 +17,8 @@ declare namespace dragula {
         delay?: boolean | number | undefined;
         mirrorContainer?: Element | undefined;
         ignoreInputTextSelection?: boolean | undefined;
+        slideFactorX?: number | undefined;
+        slideFactorY?: number | undefined;
```
Comment (andrewbranch) [inline types/dragula/v2/index.d.ts:21]: It doesn’t look like these exist in v2: https://github.com/bevacqua/dragula/tree/v2.1.2

**Review by andrewbranch — CHANGES_REQUESTED** [review]

**Comment by ValerioEmanuele** [comment]
Hi @andrewbranch ,

I've addressed the feedback you provided:

1. Regarding your comment on the properties not existing in v2 ("It doesn’t look like these exist in v2: https://github.com/bevacqua/dragula/tree/v2.1.2"), I have removed those properties from v2 as suggested.
2. Concerning your note about the package.json change ("Looks good except for the package.json change"), I have reverted it to the previous version.

Please let me know if there's anything else that needs to be adjusted.

Thanks!

File: types/dragula/v2/index.d.ts (hunk 2 above)
Comment (ValerioEmanuele) [reply types/dragula/v2/index.d.ts]: removed

**Review by andrewbranch — APPROVED** [review]
Thanks!

**Comment by ValerioEmanuele** [comment]
Ready to merge
